webpack.config.js

```diff
@@ -11,12 +12,11 @@ module.exports = {
                 test: path.resolve(__dirname, "src/calculator.js"),
                 use: [
                     // Export the Desmos global var
-                    "exports-loader?window.Desmos",
-                    // Set module.exports=undefined to work around Webpack chomping 'define'
-                    // https://github.com/webpack/webpack/issues/386
-                    // Set this=>window to fake closure context as window
-                    // Make jQuery available "globally" for both Desmos & MathQuill
-                    "imports-loader?undefined=>module.exports=undefined,this=>window,jQuery=jquery,window.jQuery=jquery"
+                    "exports-loader?Desmos",
+                    // Set module.exports=undefined and exports=undefined because Desmos has
+                    // dependencies that use UMD preludes that get confused when they are present.
```

Comment:
A couple more words about this--we already have some code that attempts to detect and correct this problem, but it does it by looking at `window.module` and `window.exports`. That check isn't working properly here, because webpack is putting `module` and `exports` on closured-in function parameters instead of on window. I think this should be a fairly easy fix for us.